bibliopixel/layout/geometry/matrix.py

```diff
@@ -61,7 +61,64 @@ def gen_matrix(dx, dy, serpentine=True, offset=0,
     return result
 
 
-def pixel_positions_from_matrix(coord_map):
+DEFAULT_CONFIG = {
+    "serpentine": True,
+    "rotation": Rotation.ROTATE_0,
+    "y_flip": False
+}
+
+
+def gen_matrix_multi(config, rotation=Rotation.ROTATE_0, y_flip=False):
```

Comment:
So, well, I fear I will never understand this.

How are we doing for unit tests?